README.md

```diff
@@ -1,11 +1,95 @@
 # Capsule
 
-A framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+A framework for network function development. Written in Rust, inspired by [NetBricks](https://www.usenix.org/system/files/conference/osdi16/osdi16-panda.pdf) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Quick Start](#quick-start)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [Contact](#contact)
+* [Maintainers](#maintainers)
+* [License](#license)
 
 ## Overview
 
-The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe.
+The goal of `Capsule` is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe. By marrying `DPDK` with `Rust` Programming Language, `Capsule` offers
+
+* a fast packet processor that uses minimum number of CPU cycles.
+* a rich packet type system that guarantees memory-safety and thread-safety.
+* a concise framework that makes complex applications easy to write.
+
+## Quick Start
+
+The easiest way to get started developing `Capsule` applications is to use the `Vagrant` [virtual machine](https://github.com/capsule-rs/sandbox/blob/master/Vagrantfile) and the `Docker` [sandbox](https://hub.docker.com/repository/docker/getcapsule/sandbox) we provide. The sandbox is preconfigured with all the necessary tools and libraries for `Capsule` development.
+
+* [`DPDK` 18.11](https://doc.dpdk.org/guides-18.11/rel_notes/release_18_11.html)
+* [`Clang`](https://clang.llvm.org/) and [`LLVM`](https://www.llvm.org/)
+* [`Rust` 1.42](https://blog.rust-lang.org/2020/03/12/Rust-1.42.html)
+* [`rr`](https://rr-project.org/) 5.3
+
+You need to first install [`Vagrant`](https://www.vagrantup.com/) and [`VirtualBox`](https://www.virtualbox.org/) on your system. And install the following `Vagrant` plugins,
+
+```
+host$ vagrant plugin install vagrant-reload vagrant-disksize vagrant-vbguest
+```
+
+Then you can clone the sandbox repository, start and ssh into the vagrant VM,
```

Comment:
mainly, remove `you`. 

> Then, clone our sandbox repository, start up and ssh into our vagrant VM, like so: 

*Note*: our sounds good in some places here.

Reply:
I will remove them from anywhere that's giving an instruction. but in general, trying to avoid using `you` as a pronoun makes regular sentences sound more awkward.

Reply:
yeah. just try your best.

Reply:
mostly "you" free.